src/rest_framework_dso/renderers.py

```diff
@@ -3,8 +3,9 @@
 
 import orjson
 from gisserver.geometries import WGS84
+from rest_framework import renderers as orig
```

Comment:
Misschien `rest_renderers` ipv `orig` voor extra leesbaarheid?

Reply:
goede, ik heb het aangepast naar `from rest_framework import renderers`. Dat patroon gebruiken we ook elders in het project.